orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -546,6 +550,20 @@ void addObjectMethodsForTraining(py::module& m, ExecutionProviderRegistrationFn
         return false;
 #endif
   });
+  m.def("is_triton_enabled", []() -> bool {
+#ifdef ENABLE_TRITON
+    return true;
+#else
+        return false;
```

Comment:
indentation

Reply:
It's actually the tool from Lint Format that formatted the code to such indentation (similar code right above). Removing the spaces will cause Lint Format error. I think this is the bug from Lint Format, but we need to make the pipeline check works.